app/controllers/api/v1/businesses_controller.rb

```diff
@@ -29,6 +33,7 @@ def create
 
   # PATCH/PUT /businesses/:slug
   def update
+    authorize @business
```

Comment:
instead use  `before_action :authorize_object,  only: %i[update destroy]`

and then define the private method:

```
def authorize_business
    authorize @business
end
```

Reply:
I named it `authorize_user` since it is the user we're authorizing.